l3sys-query.sty

```diff
@@ -0,0 +1,72 @@
+
+\ProvidesPackage{l3sys-query}[2024-03-08 v0.01 LaTeX2e interface for l3sys file queries]
+
+% Copyright 2024  The LaTeX Project
+% License: LPPL 1.3c
+
+% Defines:
+
+% \QueryPWD {<result>}
```

Comment:
\GetPWD perhaps?

Reply:
yes not sure,  Originally thought of making all commands have a consistent `\Query... ` but currently we only have two anyway so hard to know  `\Get` perhaps better.

Reply:
not sure either ... was just wondering as "query" is new in and in case of `\QueryFiles` not really the right description imo.

Reply:
also PWD is print working directpry so the P is redundant with Get (or Query) but I can't think of a better name for now. It's `\TeXOSQueryCwd`  in texosquery, Nicola has consistent `\TeXOSQuery` command prefix which would suggest `\QueryCwd` query current working directory making more sense than query print working directory,  perhaps

Reply:
I went wit @wspr `\QueryWorkingDirectory `